grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcService.java

```diff
@@ -72,6 +73,8 @@
 class UnframedGrpcService extends SimpleDecoratingService<HttpRequest, HttpResponse>
         implements ServiceWithPathMappings<HttpRequest, HttpResponse> {
 
+    private static String LINE_SEPARATOR = "\n";
```

Comment:
`private static char LINE_SEPARATOR = '\n'` (though I guess inlining it is fine too)